modules/user_accounts/jsx/userAccountsIndex.js

```diff
@@ -122,9 +122,10 @@ class UserAccountsIndex extends Component {
       }
       break;
     case t('Pending Approval', {ns: 'user_accounts'}):
-      if (row['Pending Approval'] === 'Y') {
+      const pendingKey = t('Pending Approval', { ns: 'user_accounts' });
+      if (row[pendingKey] === 'Y') {
         result = <td>{t('Yes', {ns: 'loris'})}</td>;
-      } else if (row['Pending Approval'] === 'N') {
+      } else if (row[pendingKey] === 'N') {
         result = <td>{t('No', {ns: 'loris'})}</td>;
```

Comment:
Hi @SKADE2303, we need to do the same for the `Active` column. It's not getting translated so far.
<img width="1251" height="198" alt="image" src="https://github.com/user-attachments/assets/e94a47e8-b290-4b98-8200-66190df5dac7" />